loc/US/strings_db.lua

```diff
@@ -1903,19 +1903,19 @@ Unit_Description_s283="Constructs Tech 3 Air units. Buildable for a much cheaper
 
 -- SERAPHIM -- Armored Commander Unit/Upgrades
-Unit_Description_0420="Armored Commander is a combination of barracks and command center. Contains all the blueprints necessary to build a basic army from scratch. Upgradeable with combat enhancements, advanced engineering suits, resource allocation system, and teleporting."
+Unit_Description_0420="The Armored Command Unit (ACU) is a combination of barracks and command center. Contains all the blueprints necessary to build a basic army from scratch. Upgradeable with combat enhancements, advanced engineering suits, resource allocation system, and teleporting."
 Unit_Description_0421="Adds Teleportation capability. Requires considerable energy to activate."
-Unit_Description_0422="Massively increases the rate at which the ACU repairs its armour as well as providing a large HP boost."
+Unit_Description_0422="Massively increases the rate at which the ACU repairs its armour as well as providing an HP boost.\n\n+2000 Health\n+60 Regen"
 Unit_Description_0424="Increases ACU's resource generation to 17 mass per second and 2020 energy per second."
 Unit_Description_0425="Increases ACU's resource generation to 33 mass per second and 4020 energy per second."
-Unit_Description_0426="Grants Tech 2 schematic access and increases the ACU's build speed and maximum health"
-Unit_Description_0427="Replaces the Tech 2 engineering Suite. Grants Tech 3 and Experimental schematic access and further increases the ACU's build speed and maximum health"
-Unit_Description_0428="Increases main cannon damage to 850% and grants Area-Of-Effect damage radius."
-Unit_Description_0429="Emits a field which repairs nearby allies units."
-Unit_Description_0430="Enhances the Repair Field to grant faster restoration in a larger area, as well as providing a boost to the maximum HP of those units."
-Unit_Description_0431="Doubles the ACU's main weapon Rate-Of-Fire, and increases its range and that of Overcharge."
+Unit_Description_0426="Grants Tech 2 schematic access and increases the ACU's build speed and maximum health.\n\n+32 Buildpower\n+2000 Health\n+10 Regen"
+Unit_Description_0427="Grants Tech 3 and Experimental schematic access and further increases the ACU's build speed and maximum health.\n\n+58 Buildpower\n+1500 Health\n+10 Regen"
+Unit_Description_0428="Increases main cannon damage to 720% and grants Area-Of-Effect damage radius.\n\n+620 Main Weapon damage\n+2.7 Main Weapon damage radius"
+Unit_Description_0429="Emits a field which repairs nearby allied units, as well as providing a boost to the maximum HP of those units.\n\n+500 Health (ACU only)\n+10 Regen (ACU only)\nAura Radius = 30\nAura HP-buff = 10%"
+Unit_Description_0430="Enhances the Repair Field to grant faster restoration in a larger area, as well as providing a larger boost to the maximum HP of those units.\n\n+1000 Health (ACU only)\n+10 Regen (ACU only)\nAura Radius = 35\nAura HP-buff = 20%"
```

Comment:
This formats it into a nice-looking table, and includes all the information someone would want to know.
```suggestion
Unit_Description_0429=Emits a field which repairs nearby allied units, as well as providing a boost to the maximum HP of those units.\n\n+500 Health (ACU only)\n+10 Regen (ACU only)\nRegen Aura - Radius: 30, HP Buff: 10%,\n%HP Regen: 2.0%, Maximum Regen:\nT1: 10, T2: 15, T3: 25, SACU: 30, T4: 40
Unit_Description_0430="Enhances the Repair Field to grant faster restoration in a larger area, as well as providing a larger boost to the maximum HP of those units.\n\n+1000 Health (ACU only)\n+10 Regen (ACU only)\nRegen Aura - Radius: 35, HP Buff: 20%,\n%HP Regen: 3.5%, Maximum Regen:\nT1: 20, T2: 50, T3: 120, SACU: 140, T4: 240"
```
![image](https://github.com/FAForever/fa/assets/82986251/74013fb6-9f62-4320-ac15-883511969127)

Reply:
I will add the %HP Regen metric, but won't add the Regen caps for now. I think it is the least important metric of the ones displayed, and I also don't want this description to become too convoluted. There is a lot going on with this upgrade already.

Reply:
The Regen caps are very easily reached by everything except T1 and support units, so they're one of the most important stats. A complex upgrade requires a complex description so that people can fully understand it, and I don't want to see complaints like "The ilshie has 2500 HP, why am I not getting 50 regen at level 1 aura?", or 2345 regen on the Ythotha, or 164 regen on the othuum.

Reply:
I don't necessarily agree, but I have added it in for now. One problem will be the formatting, I am not happy with either of our solutions.